packages/postgres/lib/postgres_p.js

```diff
@@ -133,6 +133,9 @@ function createSqlData(connParams, query) {
   var data = new SqlData(DATABASE_VERS, DRIVER_VERS, connParams.user,
     connParams.host + ':' + connParams.port + '/' + connParams.database,
     queryType);
+  if (process.env.ENABLE_SQL_QUERY_CAPTURE) {
```

Comment:
Can you change this to `AWS_XRAY_COLLECT_SQL_QUERIES`? To be consistent with our other SDKs that have this feature